src/tests/pages/app.page.js

```diff
@@ -0,0 +1,30 @@
+export class AppPage {
```

Comment:
Stejně jako `LoginPage`, ani `AppPage` object model tu úplně nemusí být. Vzniká totiž tzv. dead code - nevyužitý kód - funkce `getToastMessage()`, `logout()` a `getCurrentUser()` se nikde v rámci /test/homework.spec.js nepoužívají.